src/libstd/rt.rs

```diff
@@ -65,3 +66,10 @@ fn lang_start(main: *const u8, argc: isize, argv: *const *const u8) -> isize {
         0
     }
 }
+
+/// Fixed frame used to clean the backtrace with `RUST_BACKTRACE=1`.
+#[no_mangle]
+#[inline(never)]
+pub fn __rust_begin_backtrace_binary(f: fn()) {
```

Comment:
https://play.rust-lang.org/?gist=e7c60682076e24a4f2ad4fe56af59de1&version=stable&backtrace=1

What do you think would be better, `#[ignore(..)]` the warning or keep it pub?